build/make-nojquery.js

```diff
@@ -0,0 +1,68 @@
+// jshint esversion:6, node:true
+
+const fs = require("fs");
+const path = require("path");
+
+const README = `# DevExtreme ASP.NET Data
+
+jQuery-free version of [devextreme-aspnet-data](https://www.npmjs.com/package/devextreme-aspnet-data)
+`;
+
+const JQ_SURROGATE_AMD = `{
+                    ajax: require("devextreme/core/utils/ajax").sendRequest,
+                    Deferred: require("devextreme/core/utils/deferred").Deferred,
+                    extend: require("devextreme/core/utils/extend").extend
+                }`;
+
+const JQ_SURROGATE_GLOBAL = `throw "This script should be used with an AMD loader"`;
+
+const AJAX_SETTINGS_SURROGATE = `{
+        cache?: boolean;
+        contentType?: any;
+        data?: any;
+        dataType?: string;
+        headers?: { [key: string]: any; };
+        method?: string;
+        password?: string;
+        timeout?: number;
+        url?: string;
+        username?: string;
+        xhrFields?: { [key: string]: any; };
+    }`;
+
+const outputPath = path.join(__dirname, "..", "js-nojquery");
+if(!fs.existsSync(outputPath))
+    fs.mkdirSync(outputPath);
+
+const packageJSON = require("../package.json");
+[ "main", "types", "devDependencies", "scripts" ].forEach(i => delete packageJSON[i]);
```

Comment:
Maybe it's better to extract the necessary blocks instead of removing redundant ones?